packages/editor/src/client/global.less

```diff
@@ -7,19 +7,19 @@
   &-row.approved:not(&-row-header),
   &-row.approved &-cell-group,
   &-row.approved &-cell {
-    background: rgb(225, 248, 222);
+    background: #e1f8de;
```

Comment:
are those entries still needed? And if yet where?

Reply:
@roosnic1 this is for the colors on the comment list. I thought it's better if it's the same colors throughout the editor.

Reply:
👍🏾